data/102/079/921/102079921.geojson

```diff
@@ -93,13 +93,14 @@
     "wof:breaches":[],
     "wof:concordances":{
         "pl-gugik":"3064",
-        "wd:id":"Q6680284"
+        "wd:id":"Q268",
+        "wk:page":"Pozna\u0144"
```

Comment:
Going forward past this PR...

Generally speaking I'm only in favor of adding `wd:id` values and not `wk:page` values as that an be a lookup from `wd:id` list of related pages from their service, and `wk:page` implies English and we should definitely _not_ create new namespaces for all localized Wikipedia content. But it's fine here.